packages/manager/cypress/e2e/core/cloudpulse/dbaas-widgets-verification.spec.ts

```diff
@@ -191,40 +187,67 @@ describe('Integration Tests for DBaaS Dashboard ', () => {
     ui.autocomplete
       .findByLabel('Dashboard')
       .should('be.visible')
-      .type(`${dashboardName}{enter}`)
-      .should('be.visible');
+      .type(dashboardName);
+
+    ui.autocompletePopper
+      .findByTitle(dashboardName)
+      .should('be.visible')
+      .click();
 
     // Select a time duration from the autocomplete input.
     ui.autocomplete
       .findByLabel('Time Range')
       .should('be.visible')
-      .type(`${timeDurationToSelect}{enter}`)
-      .should('be.visible');
+      .type(timeDurationToSelect);
+
+    ui.autocompletePopper
+      .findByTitle(timeDurationToSelect)
+      .should('be.visible')
+      .click();
 
-    //Select a Engine from the autocomplete input.
+    //Select a Database Engine from the autocomplete input.
     ui.autocomplete
       .findByLabel('Database Engine')
       .should('be.visible')
-      .type(`${engine}{enter}`)
-      .should('be.visible');
+      .type(engine);
 
-    // Select a region from the dropdown.
-    ui.regionSelect.find().click().type(`${region}{enter}`);
+    ui.autocompletePopper
+       .findByTitle(engine)
+        .should('be.visible')
+        .click();
 
-    // Select a resource from the autocomplete input.
+    //  Select a region from the dropdown.
+    ui.regionSelect.find().click();
+    ui.regionSelect
+      .findItemByRegionId(mockRegion.id, [mockRegion])
+      .should('be.visible')
+      .click();
+
+    // Select a resource (Database Clusters) from the autocomplete input.
     ui.autocomplete
       .findByLabel('Database Clusters')
       .should('be.visible')
-      .type(`${clusterName}{enter}`)
-      .click();
-    cy.findByText(clusterName).should('be.visible');
+      .type(clusterName);
 
-    //Select a Node from the autocomplete input.
+    ui.autocompletePopper.findByTitle(clusterName).should('be.visible').click();
+
+    // Select a Node from the autocomplete input.
     ui.autocomplete
       .findByLabel('Node Type')
       .should('be.visible')
       .type(`${nodeType}{enter}`);
 
+    // Expand the applied filters section
+    ui.button.findByTitle('Filters').should('be.visible').click();
+
+    // Verify that the applied filters
+    cy.get('[data-testid="applied-filter"]').within(() => {
+      cy.get(`[data-qa-value="Database Engine ${engine}"]`);
+      cy.get(`[data-qa-value="Region US, Chicago, IL"]`);
+      cy.get(`[data-qa-value="Node Type ${nodeType}"]`);
+      cy.get(`[data-qa-value="Database Clusters ${databaseMock.label}"]`);
```

Comment:
@agorthi-akamai , we should check for the element to be visible and have the expected value